index.html

```diff
@@ -250,15 +252,16 @@ <h3>Ecosystem Overview</h3>
         </dd>
         <dt><a>verifier</a></dt>
         <dd>
-A role an <a>entity</a> may perform by receiving one or more
-<a>verifiable presentations</a> for processing.
+A role an <a>entity</a> may perform by requesting and receiving a
+<a>verifiable presentation</a> that proves the holder possesses the required verifiable credentials with certain characteristics.
 Examples of verifiers include employers, security personnel, and websites.
         </dd>
         <dt><a>identifier registry</a></dt>
         <dd>
-A role a system may perform by mediating the creation and verification of
-<a>subject</a> identifiers. Examples of identifier registries include
-corporate employee databases, government ID databases, and distributed ledgers.
+A role a system may perform by mediating the creation and verification of <a>issuer</a> identifiers,
+keys and other relevant data required to use verifiable credentials. Some configurations may require identifiers
+for subjects but leads to subjects being correlated. Examples of such data repositories include trusted databases,
```

Comment:
`Some configurations may require identifiers
for subjects but leads to subjects being correlated.` => `Some configurations may require correlatable identifiers for subjects.`

Reply:
I don't see the difference in meaning or implications.